accessibility/index.html

```diff
@@ -0,0 +1,556 @@
+					<p>A technical standard made of Four Principles with Twelve Guidelines containing testable Sucess Criteria</p>
+					<p>We will refer to the Web Content Accessibility Guideliness and Success Crtieria in the upcoming excercises</p>
+					<small><a href="http://www.w3.org/TR/UNDERSTANDING-WCAG20/Overview.html" target="_blank"><em>Understanding Web Content Accessibility Guidelines 2.0</em></a><br/>W3C - Web Accessibility Initiative</small>
+				</section>
+
+				<!-- WCAG 2.0 END -->
+            
+                <!-- SUCCESS CRITERIA EXAMPLES START -->
+            
+                <section>
+                    <h2>WCAG <span class="gdi">Success Criteria</span> Examples</h2>
+                    <pre><code data-trim contenteditable> 
+                    
+<!-- SC 1.3.1 -->
+ DOCTYPE html lang="en"
+ 
+ <!-- SC 1.3.1 - SC 2.4.1 - SC 2.4.6 - SC 2.4.10 -->
+ <h1>Page Heading</h1> 
+ 
+ <!-- SC 1.3.1 - SC 2.4.1 - SC 2.4.6 - SC 2.4.10 -->
+ <h2>Join our mailing list</h2>
+
+ <!-- SC 1.3.1 - SC 3.3.3 - SC 4.1.2 -->
+ <label for="email">Please enter your email address</label> 
+ <input type="text" name="email" placeholder="Please enter your email address">
+
+                    </code></pre>
+                    
+                </section>
+            
+                <!-- SUCCESS CRITERIA EXAMPLES END -->
+            
+             <!-- ERROR -->
+            
+            <section>
+                        <small>(SC 1.3.1 - SC 2.4.1 - SC 2.4.6 - SC 2.4.10)</small>
+                        <h2>Looks <span class="gdi">Complicated</span></h2>
+                        <h2>It Isn't!!</h2>
+						<small><a href="http://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv.html" target="_blank">Understanding WCAG Sucess Criteria</a></small>
+					</section>
+                    <aside class="notes">
+				    How to easily use the WCAG Success Criteria - Understanding WCAG Sucess Criteria
+                    </aside>
+				</section>
+
+				<!-- Exercise 04 START-->
+
+				<section>
+					<h3>Exercise 4</h3> 
+					<p>Correct errors produced by automated test results</p>
+                    <small><a href="https://jsbin.com/liyewotacu/edit?html,css,output" target="_blank">Modify HTML from Exercise Two</a></small>		
+				</section>
+
+				<!-- Exercise 04 END -->
+
+				<!-- WHAT IS ARIA START -->
+				
+				<section>
+					<h3>Accessible Rich<br/>Internet Applications (ARIA 2.0)</h3>
+					<p>A set of accessibility attributes especially suited to HTML</p>		
+				</section>
+
+				<!-- WHAT IS ARIA END -->
+
+				<!-- ARIA ROLES START -->
+				
+				<section>
+					<h2>ARIA Roles</h2>
+                    <p><span class="gdi"><strong>Roles</strong></span> define the <strong>type</strong> of element or widget presented to the accessibiltiy API</p>
+                    <p>For example you can give a div a role="link" or role="button". The div will now be announced as a link or a button by a screen reader.</p>
```

Comment:
Can you add another slide that is a list of popular "roles" and may contain useful information for completing the next exercise?